akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -227,6 +228,7 @@ object Attributes {
 
   val none: Attributes = Attributes()
 
+  val verboseLogging: Attributes = Attributes(VerboseLogging)
```

Comment:
good, needs some docs on the method as well as on the object.
Docs could be something like "enables verbose logging in stages that chose to support it. Such logging can be useful when debugging unexpected behaviour in element passing, however should not be used in day-to-day use as it likely will result in too vast amounts of logs" or something like that?